### ISSUE TITLE
Group tree not pronounced by screen reader (Orca)

### ISSUE THREAD

**Issue by phoerious** [keepassxreboot/keepassxc#8455]
Forwarded user feedback: Orca does not read the contents of the group tree when tabbing from the search field to the root group.

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - 2.7.1

[NOTE]: # ( Pick choices based on your environment )
Operating System: Linux

**Comment by droidmonkey** [comment]
This also happens with Microsoft Narrator. For whatever reason it always prefers to read the first entry title and not the focused group title.

**Comment by cannona** [comment]
I'm also experiencing this issue with Orca on Linux, as well as NVDA (a free and open source screen reader for Windows), as well as with the commercial screen reader JAWS for Windows.

The treeview simply does not act in an expected manner. I.E. left arrow moves to the parent/collapses the current branch, right arrow expands or enters child branches, up and down arrow takes you through the children, etc. The level of each element in the tree should also be communicated to the assistive technology: level 1 for the top level element, 2 for children, 3 for grandchildren, etc.

**Comment by droidmonkey** [comment]
Please try our snapshot build that uses qt6. https://snapshot.keepassxc.org

**Comment by cannona** [comment]
That works a lot better. Not perfect, but definitely usable with NVDA on Windows.
